lib/vsc/mympingpong/tools.py

```diff
@@ -45,8 +45,10 @@ def hwlocmap():
 
     Returns a dict that maps the absolute Processor Unit ID to its socket-id and its core-id
     """
-    xmlout = tempfile.mkstemp(prefix="hwloc-xml-", suffix=".xml")
-
+    # Only need a filename
+    (fh, xmlout) = tempfile.mkstemp(prefix="hwloc-xml-", suffix=".xml")
+    os.close(fh)
```

Comment:
I would like it more  if we could pass the file handle along, instead of closing it and reopening it